pkg/generator/forms.go

```diff
@@ -71,6 +71,7 @@ func GenerateForms() TileFormsSections {
 			GenerateDatabaseForm(),
 			GenerateEnableDisableForm(),
 			GenerateRoleWhitelistForm(),
+			generateCompatibilityForm(),
```

Comment:
Not related to your changes, but are the other `Form` functions exported unnecessarily?